### PR TITLE
fix dissapearning media in edit page form

### PR DESCRIPTION
**Pull request by BartoszWojdalowicz**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

**Comment by BartoszWojdalowicz** [comment]
i tested it manually and everything looks good, no errors in chrome console or profiler.

File: src/Resources/public/js/bitbag-media-autocomplete.js [hunk 1]
```diff
@@ -79,9 +79,6 @@ function htmlToString(item) {
                 }
               });
             }
-            window.setTimeout(function () {
```
Comment (christopherhero) [inline src/Resources/public/js/bitbag-media-autocomplete.js:82]: Please check https://github.com/Sylius/Sylius/pull/12324 (maybe you still need to set up the data)

Comment (BartoszWojdalowicz) [reply src/Resources/public/js/bitbag-media-autocomplete.js:82]: unfortunately with this code problem still exist. when i'm on list of pages i have thumbnails if  has been added to the page, but when i go to edit page, image doesn't show in form.

Comment (christopherhero) [reply src/Resources/public/js/bitbag-media-autocomplete.js:82]: This autocomplete file is not equal to the generic file `sylius-autocomplete.js` on the edit page does not load. 
You should set selected mutators. 
-
in the my pull request for syllius (https://github.com/Sylius/Sylius/pull/12324) I described the PR description what was the problem check this description :)

if everything is correct, ignore :)

**Pull request merged**